File: CAMPUSMONK/L5_change_case_all_characters.py
s="Bibhakar Paul"
o=""
for i in s:
    if i>='a' and i<='z':
        o+=chr(ord(i)-32)
    elif i==' ':
        o+=chr(32) 
    else:
        o+=chr(ord(i)+32)
print(o)

File: CAMPUSMONK/L5_change_case_all_characters.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    string s="Bibhakar Paul";
    for(int i=0;i<s.size();i++)
    {
        if(s[i]>='a' && s[i]<='z')
        {
            s[i]-=32;
        }
        if(s[i]==' ')
        {
            s[i]*=1;
        }
        else
        {
            s[i]+=32;
        }
    }
    cout<<s<<endl;
}
